### DISCUSSION TITLE
静态资源文件链接外置

### DISCUSSION THREAD

**Discussion by Gahotx** [Qexo/Qexo#126] [Ideas]
如题，因为jsd现在速度很慢，我看qexo用到的静态文件都是用的jsd加速的qexo-static，希望能够将静态资源文件链接外置，可以让用户自定义什么[七牛云](https://www.staticfile.org/)，[字节跳动](https://cdn.bytedance.com/)等公共库，或者自己的存储桶来加速qexo

**Comment by am-abudu** [comment]
修改高级设置中的`CDN_PREV`项
